internal/Position.go

```diff
@@ -1,7 +1,14 @@
 package internal
 
-type Position struct {
+type Position interface{}
```

Comment:
I didn't get the purpose why do you declare and then embed the empty interface?

Reply:
+